aws/resource_aws_route53_resolver_endpoint.go

```diff
@@ -0,0 +1,341 @@
+package aws
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/route53resolver"
+	"github.com/hashicorp/terraform/helper/hashcode"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+)
+
+const (
+	Route53ResolverEndpointStatusDeleted = "DELETED"
```

Comment:
this probably doesn't need to be exported

Reply:
correct, it doesn't